pyproject.toml

```diff
@@ -44,9 +44,10 @@ test = [
 ]
 
 demo = [
-    "websockets==12.0",
-    "flask==2.3.2",
-    "flask-cors==4.0.0"
+    "requests",
```

Comment:
Loosened these to be a little more cross-compatible, unless we really do need very specific versions to get this to work?

Reply:
This is fine. Thank you!